scripts/branch_rename_migration.py

```
@@ -0,0 +1,251 @@
+            else:
+                print(f"✗ Failed to delete local branch '{branch}'")
+        
+        # Delete remote branches marked for deletion
+        for branch in remote_branches_to_delete:
+            if delete_remote_branch(f"origin/{branch}"):
+                print(f"✓ Deleted remote branch 'origin/{branch}'")
+            else:
+                print(f"✗ Failed to delete remote branch 'origin/{branch}'")
+        
+        # Rename remote branches
+        for old_name, new_name in remote_branches_to_rename:
```

Ruff (0.14.2)

Loop control variable `new_name` not used within loop body

(B007)